tests/zfs-tests/tests/functional/no_space/enospc_003_pos.ksh

```diff
@@ -51,7 +51,11 @@ log_onexit cleanup
 
 log_assert "ENOSPC is returned on pools with large physical block size"
 
-log_must zpool create $TESTPOOL1 -o ashift=13 $DISK_LARGE
+if is_freebsd; then
+	log_must zpool create -o ashift=13 $TESTPOOL1 $DISK_LARGE
+else
+	log_must zpool create $TESTPOOL1 -o ashift=13 $DISK_LARGE
+fi
```

Comment:
Why can't we use the `log_must zpool create -o ashift=13 $TESTPOOL1 $DISK_LARGE` on both platforms?